### PR TITLE
Added bitshuffle

### PR DESCRIPTION
**Pull request by xmnlab**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/bitshuffle/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kiyo-masui/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: c3f4461d8013e3d9db0d58defec77b143164652de505a1fba3df088eaa19be2f
+
+build:
+  noarch: python
+  number: 0
```
Comment (chrisburr) [inline recipes/bitshuffle/meta.yaml]: If you can [you should use pip here](http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip#use-pip).

File: recipes/bitshuffle/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+requirements:
+  build:
+    - cython
```
Comment (chrisburr) [inline recipes/bitshuffle/meta.yaml]: If you're using `cython` you probably [need a compiler](http://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=compiler#compilers):

File: recipes/bitshuffle/meta.yaml (hunk 1 above)
Comment (xmnlab) [reply recipes/bitshuffle/meta.yaml]: sounds good! thanks @chrisburr !

File: recipes/bitshuffle/meta.yaml (hunk 2 above)
Comment (xmnlab) [reply recipes/bitshuffle/meta.yaml]: great! thanks @chrisburr ! I will try that!

**Pull request merged**